code/modules/industrial_lift/tram_lift_master.dm

```diff
@@ -123,6 +124,8 @@
 
 /datum/lift_master/tram/process(delta_time)
 	if(!travel_distance)
+		update_tram_doors(LOCK_DOORS)
```

Comment:
I don't know why they get locked and then unlocked right away, but sure I guess.